babel-watch.js

```diff
@@ -294,6 +299,18 @@ function restartApp() {
   childApp = app;
 }
 
+const delay = program.delay || 1000;
```

Comment:
Thank you.
I think maybe at least 100ms is necessary. Otherwise, when more than one files are generated at the same time, restarting will be horrible.
What about 100ms?